engine/class_modules/paladin/sc_paladin.cpp

```diff
@@ -1078,6 +1185,7 @@ struct ashen_hallow_t : public paladin_spell_t
     may_miss = false;
 
     add_child( damage_tick );
+    add_child( new hallowed_discernment_tick_t( p ) );
```

Comment:
This hallowed discernment tick isn't used except for setting the child action. You could send it to ashen hallow's constructor so you don't have to create it another time there.